pom.xml

```diff
@@ -47,6 +47,15 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>22</source>
+                    <target>22</target>
+                    <compilerArgs>--enable-preview</compilerArgs>
+                </configuration>
+            </plugin>
```

Comment:
Remove these changes, right now all checks are failing because of it! Also, in the future, do not create PR when you get this (If you can't manage to repair errors by yourself, ask for help on Mate platform):
![image](https://github.com/mate-academy/jv-oop/assets/140905715/03bca39c-abb3-425d-9150-29598ff880d2)

Reply:
When you repair this, you will see a couple of errors because of the checkstyle validation, adjust your code accordingly.